sentry_sdk/integrations/django/__init__.py

```diff
@@ -438,7 +454,7 @@ def _got_request_exception(request=None, **kwargs):
 
         if request is not None and integration.transaction_style == "url":
             with hub.configure_scope() as scope:
-                _attempt_resolve_again(request, scope)
+                _attempt_resolve_again(request, scope, integration.transaction_style)
```

Comment:
this only runs in the `transaction_style="url"` case so not sure you need the extra argument here

Reply:
I rather keep the argument so `_attempt_resolve_again` is not assuming outside state.